### ISSUE TITLE
Sync-watch: Manual episode changes are not synced

### ISSUE THREAD

**Issue by Skehmatics** [streamaserver/streama#758]
### Issue description
When using the synchronized watch feature, manual episode changes are not synced to other users.

### Steps to Reproduce

1. Login to Streama for two users (use an incognito / private window for one to keep states seperate)
2. On one window, start playing a show / TV series. 
3. Click the new session button in the control bar (second from the right)
4. Copy the link from window one and paste it into window two
5. Force an episode change, either by
   a. using the episode selector (third from the right)
   b. seeking towards the last 30 seconds of an episode and pressing the "play now" button

### Expected Behaviour

Episode on other window changes in tandem with window that initiated the change

### Actual Behaviour

If playback is paused, it resumes on the other window, otherwise the other is unaffected.

### Environment Information

- **Operating System**: Linux (Ubuntu Server 18.04LTS)
- **Streama version**: v1.6.0-FINAL
- **Custom streama build, describe customizations or provide link to fork (If Applicable)**: N/A
- **Container Version (If Applicable):** N/A

**Comment by CardealRusso** [comment]
this is urgent

**Comment by dularion** [comment]
Yeah that makes sense to implement. I will try to get to it soon.
